downstream/modules/platform/ref-eda-controller-variables.adoc

```diff
@@ -0,0 +1,45 @@
+
+[id="event-driven-ansible-controller_{context}"]
+= {EDAcontroller} variables
+
+[cols="50%,50%",options="header"]
+|====
+| *Variable* | *Description* 
+| *`automationedacontroller_admin_username`* | Username used by django to identify and create the admin superuser in {EDAcontroller}.
+
+Default = admin
+| *`automationedacontroller_admin_email`* | Email address used by django for the admin user for {EDAcontroller}. 
+
+Default = admin@example.com
+| *`automationedacontroller_disable_https`* | Boolean flag to disable HTTPS {EDAcontroller}. 
+
+Default = false
+| *`automationedacontroller_disable_hsts`* | Boolean flag to disable HSTS {EDAcontroller}. 
+
+Default = false
+| *`automationedacontroller_user_headers`* | List of additional nginx headers to add to {EDAcontroller}'s nginx configuration. 
+
+Default = empty list
+| *`automationedacontroller_nginx_tls_files_remote`* | Boolean flag to specify whether cert sources are on the remote host (true) or local (false). 
+
+Default = false
+| *`automationedacontroller_allowed_hostnames`* | List of additional addresses to enable for user access to {EDAcontroller}.
+
+Default = empty list
+| *`automationedacontroller_controller_verify_ssl`* | Boolean flag used to verify Automation Controller's web certificates when making calls from {EDAcontroller}. Verified is true; not verified is false. 
+
+Default = false
+//Add this variable back for the next release, as long as approved by development.
+//| *`automationedacontroller_websocket_ssl_verify`* | 
+//SSL verification for the Daphne websocket used by podman to communicate from the pod to the host. Default is false to disable SSL connection as verified
+
+//Default = false
+| *`automationedacontroller_gunicorn_workers`* | Number of workers for the API served through gunicorn.
+
+Default = (# of cores or threads) * 2 + 1
```

Comment:
This looks like ( ...) x3, would 2*( ...) +1 be clearer?

Reply:
Order of operations its fine, but I could understand how someone could read it in their head as * 3. If you moved the 2 to the front that'd be fine: `2 * (# of cores of threads) + 1` just as long as 2 is spaced enough that they can see it and not overlook.

Reply:
Hi @ianf77. I think I need to check with James Marshaller (Installation SME) to see if he agrees with that proposal.

Reply:
Okay, he is agreeable to the update. Will update in a bit.